pkg/synthesis/createK8sResources.go

```diff
@@ -45,128 +44,13 @@ func createK8sResources(model *AbstractModelSyn, outDir string) error {
-	pol := &networking.NetworkPolicy{}
-	pol.TypeMeta.Kind = "NetworkPolicy"
-	pol.TypeMeta.APIVersion = k8sAPIVersion
-	pol.ObjectMeta.Name = name
-	pol.ObjectMeta.Annotations = map[string]string{"description": description}
-	return pol
-}
-
-func toSelectorsAndPorts(p *symbolicexpr.SymbolicPath) (srcSelector, dstSelector *meta.LabelSelector,
-	ports []networking.NetworkPolicyPort, empty bool) {
-	srcSelector = toSelector(p.Src)
-	dstSelector = toSelector(p.Dst)
-	ports, empty = toPolicyPorts(p.Conn)
-	return
-}
-
-var codeToProtocol = map[int]core.Protocol{netset.UDPCode: core.ProtocolUDP, netset.TCPCode: core.ProtocolTCP}
-var boolToOperator = map[bool]meta.LabelSelectorOperator{false: meta.LabelSelectorOpExists, true: meta.LabelSelectorOpDoesNotExist}
-
-func pointerTo[T any](t T) *T {
-	return &t
-}
-
-func toSelector(con symbolicexpr.Conjunction) *meta.LabelSelector {
-	selector := &meta.LabelSelector{}
-	for _, a := range con {
-		label, notIn := a.AsSelector()
-		if label != "" { // not tautology
-			req := meta.LabelSelectorRequirement{Key: label, Operator: boolToOperator[notIn]}
-			selector.MatchExpressions = append(selector.MatchExpressions, req)
-		}
-	}
-	return selector
-}
-
-func toPolicyPorts(conn *netset.TransportSet) ([]networking.NetworkPolicyPort, bool) {
-	ports := []networking.NetworkPolicyPort{}
-	tcpUDPSet := conn.TCPUDPSet()
-	if tcpUDPSet.IsEmpty() {
-		return nil, true
-	}
-	if tcpUDPSet.IsAll() {
-		return nil, false
-	}
-	partitions := tcpUDPSet.Partitions()
-	for _, partition := range partitions {
-		protocols := partition.S1.Elements()
-		portRanges := partition.S3
-		for _, portRange := range portRanges.Intervals() {
-			var portPointer *intstr.IntOrString
-			var endPortPointer *int32
-			if portRange.Start() != netp.MinPort || portRange.End() != netp.MaxPort {
-				port := intstr.FromInt(int(portRange.Start()))
-				portPointer = &port
-				if portRange.End() != portRange.Start() {
-					//nolint:gosec // port should fit int32:
-					endPort := int32(portRange.End())
-					endPortPointer = &endPort
-				}
-			}
-			for _, protocolCode := range protocols {
-				ports = append(ports, networking.NetworkPolicyPort{
-					Protocol: pointerTo(codeToProtocol[int(protocolCode)]),
-					Port:     portPointer,
-					EndPort:  endPortPointer})
-			}
-			if slices.Contains(protocols, netset.TCPCode) && slices.Contains(protocols, netset.UDPCode) {
-				ports = append(ports, networking.NetworkPolicyPort{Protocol: pointerTo(core.ProtocolSCTP), Port: portPointer, EndPort: endPortPointer})
-			}
-		}
-	}
-	return ports, false
-}
-
 // ///////////////////////////////////////////////////////////////////////////////
 func toPods(model *AbstractModelSyn) []*core.Pod {
 	pods := []*core.Pod{}
 	for _, vm := range model.vms {
 		pod := &core.Pod{}
 		pod.TypeMeta.Kind = "Pod"
-		pod.TypeMeta.APIVersion = k8sAPIVersion
+		pod.TypeMeta.APIVersion = "v1"
 		pod.ObjectMeta.Name = vm.Name()
 		pod.ObjectMeta.UID = types.UID(vm.ID())
```

Comment:
no need to assign pod UID (it's auto generated when applied in k8s cluster).

Reply:
fixed